concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1779,7 +1779,11 @@ jobs:
                       PREFIX_JOB: "true"
                   EOF
 
-                  cf push
+                  # We only push if not running so stats are not disrupted
+                  # during deployments
+                  if ! cf app graphite-nozzle | grep started; then
```

Comment:
I don't think this implementation is correct because subsequent changes to the nozzle application will never be deployed.

Can we use blue-green here to prevent downtime like these stories for the healthcheck app? Or will it result in duplicate metrics?

- [SPIKE: Which zero-downtime plugin for cf push](https://www.pivotaltracker.com/story/show/124843741)
- [Zero-downtime deploy of healthcheck app](https://www.pivotaltracker.com/story/show/121934253)